labs/electronic-toll-collection-system.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Transparency in the Interoperability of the National Electronic Toll(ETC) network
+[//]: # (parent: Labs)
+---
+# Lab Name
+Transparency in the Interoperability of the National Electronic Toll(ETC) network
+
+# Short Description
+Hybrid Blockchain Network that seeks the route of Transparency, Traceability and Correct collection Payment between operators at Electronic Toll Collection(ETC)
+
+# Scope of Lab
+Derived from the current technical situation, the development of an interface capable of being able to insert each of the crossings made at the toll booths through electronic tolls in a private blockchain network is proposed, where in the same way it can send data of control figures to a network. Once the previous steps have been carried out, you can immediately filter and determine which operator the toll belongs to, complying with the interoperability process, leaving SFTP servers out, and adapting a queuing system that can comply with fault tolerance.
+
+In order to comply with transparency in the transactions carried out and if it is necessary to request information directly from the private network to reconcile information that the regulatory entity may question, where the private blockchain will be its main source of information, NEAR blockchain will be used. public for the issuance of delivery document receipt of requested information, generating document as NFT, and can be used as unique proof, using IPFS as a file storage system, where in the same way the integrity of the generated document can be counted on.
```

Comment:
This paragraph does not read properly. The second sentence starts with "public for the issuance of delivery document..." It seems there is something missing in the description.